### PR TITLE
Compatibility with DESI Python environments on NERSC

### PR DESCRIPTION
**Pull request by dli-astro**
1) A new module (gaussian.py) and some changes to phot.py and util.py to work around the API changes in phtoutils.
2) A few expired links to aux files have been updated.
3) New functions in concat_ccd.py and a new script (run_concad_ccds_nersc.py) have been added to generate the daily summary files faster on NERSC.

### CODE REVIEW

**Comment by schlafly** [comment]
Great to see this moving!  I apologize, but I've lost track a bit: can you summarize the current status of the GFA processing at NERSC?  Thanks!

**Comment by weaverba137** [comment]
@dli-astro has done most of the work here, but things are
moving quickly.

1. The processing runs fine at NERSC. There are a few weeks of processed files to look at in `/global/cfs/cdirs/desi/users/danli/GFA`. I would encourage you to compare those to the files downloaded from KPNO.
2. Yesterday I set up the `desi` collaboration account to run the processing, testing this branch.
3. More work is needed to make the package compatible with `desiInstall`, but that's a separate, future PR.

During this process, we discovered that the numerical results depend on the version of scipy, and apparently KPNO has a *very* old version. At NERSC we will simply use the version in desiconda, even if that means small numerical differences.

**Comment by weaverba137** [comment]
PS, this morning we had a successful test using both the `desi` collaboration account and scron to automate the processing. Currently, the intermediate processing is triggered at 14:00 UTC (07:00 MST) and the summary file creation at 17:00 UTC (10:00 MST). We can play with those times a little bit, but it does take about 2 hours to process a full night, so the 3-hour gap is being conservative.

We could trigger the intermediate processing at the *start* of the night; it will wait and process exposures as they arrive.  I will go ahead and test that mode.

**Comment by schlafly** [comment]
That's great to hear.  Yes, we should try to compare the KPNO / NERSC versions; I'll advertise on #survey-ops.

**Comment by weaverba137** [comment]
@dli-astro, @schlafly, I'm going to merge this PR and then update the package infrastructure to be more compatible with DESI packages installed at NERSC.
